python/lsst/ts/mtdometrajectory/dome_trajectory.py

```diff
@@ -501,11 +515,18 @@ async def follow_target(self):
                 telescope_target=self.telescope_target,
                 next_telescope_target=self.next_telescope_target,
             )
-            if desired_dome_azimuth is not None:
+            if desired_dome_azimuth is not None and not (
+                math.isnan(desired_dome_azimuth.position)
+                or math.isnan(desired_dome_azimuth.velocity)
```

Comment:
Are you missing a `not` here?  In any case, consider using `math.isfinite` instead.

Reply:
I rephrased these lines even though they were correct. Note the parenthesis following `not` and the `or` within the parenthesis. I replaced it with
```
            desired_dome_azimuth = self.algorithm.desired_dome_azimuth(
                dome_target_azimuth=dome_target_azimuth,
                telescope_target=self.telescope_target,
                next_telescope_target=self.next_telescope_target,
            )
```
which is clearer and less ambiguous.